src/licensedcode/models.py

```diff
@@ -1211,7 +1216,12 @@ def load_rules(rules_data_dir=rules_data_dir, with_checks=True, is_builtin=True)
                 space_problems.append(rule_file)
 
             try:
-                yield Rule.from_file(rule_file=rule_file)
+                rule = Rule.from_file(rule_file=rule_file)
+                if rule.is_deprecated and ignore_deprecated:
```

Comment:
IMHO we could mirror https://github.com/nexB/scancode-toolkit/blob/7ee80d25d8b443c94b2eeee5e8ff3dc3c921cb44/src/licensedcode/models.py#L877 or update things in both place:
```suggestion
                if not with_deprecated and rule.is_deprecated:
```

Reply:
Doing this, thanks!